pepper-apis/src/main/resources/db-changes/seed/DDP-7182-add-unique-value-validation-type.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="sampath" id="20220207-add-unique-value-validation-type">
+        <insert tableName="validation_type">
+            <column name="validation_type_code" value="UNIQUE_VALUE"/>
+        </insert>
+
+        <insert tableName="i18n_validation_msg_trans">
+            <column name="validation_type_id" valueComputed="(select validation_type_id from validation_type where validation_type_code = 'UNIQUE_VALUE')"/>
+            <column name="language_code_id" valueComputed="(select language_code_id from language_code where iso_language_code = 'en')"/>
+            <column name="translation_text" value="Value should be unique with the study question answers."/>
+        </insert>
+        <insert tableName="i18n_validation_msg_trans">
+            <column name="validation_type_id" valueComputed="(select validation_type_id from validation_type where validation_type_code = 'UNIQUE_VALUE')"/>
+            <column name="language_code_id" valueComputed="(select language_code_id from language_code where iso_language_code = 'es')"/>
+            <column name="translation_text" value="El valor debe ser único con las respuestas a las preguntas del estudio."/>
+        </insert>
+    </changeSet>
+
+</databaseChangeLog>
```

Comment:
Is this file necessary? It looks like it's in the hierarchy prior to the Maven module refactor.

Reply:
oops. Thank you.
I deleted it and looks like I added it back.